autonomi/src/client/mod.rs

```diff
@@ -28,6 +28,7 @@ pub use high_level::files;
 pub use high_level::vault;
 
 pub mod address;
+pub mod key_derivation;
```

Comment:
not seeing this to be used by other mod?

Reply:
Yes not used just yet but might prove to be useful for key management as Happybeing noted